src/front-end/src/components/Login.js

```diff
@@ -124,7 +124,7 @@ class Login extends Component {
             <p>
               <a
                 className="loginlink"
-                href="http://localhost:3000/ForgotPassword
+                href="http://localhost:3000/forgot-password
```

Comment:
To fix the issue of not being redirected to the correct page when the 'Forgot your password?' link is clicked: